git/workflow/workflow/workflow-topic-branches.md

```diff
@@ -19,7 +19,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows){website}'
+  - '[Git Branching Official Documentation](http://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows){website}'
```

Comment:
Maybe use the `documentation` resource type here? [Check out all of them](https://github.com/enkidevs/curriculum/wiki/Insight-Documentation#links)